src/loader.js

```diff
@@ -88,22 +88,26 @@
         return entry.module;
       });
 
+    // @ifdef STRICT
```

Comment:
A custom syntax worries me a little as it ties us to a specific pre-processor. While I know a global if statement is uglier, the nice thing is that it will be compatible with any preprocessor tooling in future avoiding locking us into a specific approach.